README.md

```diff
@@ -20,40 +20,43 @@
 ## Third Party Dependencies
 
 - snappy
-- lz4
 - zlib
 - thrift 0.7+ [install instructions](https://thrift.apache.org/docs/install/)
 - googletest 1.7.0 (cannot be installed with package managers)
```

Comment:
Should we provide a link to SO: http://stackoverflow.com/questions/13513905/how-to-setup-googletest-as-a-shared-library-on-linux ?